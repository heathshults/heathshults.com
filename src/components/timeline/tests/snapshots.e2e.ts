import { snapIt } from '../../../test';

const component = 'timeline';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('without items', '<hs-timeline></hs-timeline>');
    snap(
      'with items',
      `<hs-timeline>
        <hs-timeline-item>Item 1</hs-timeline-item>
        <hs-timeline-item>Item 2</hs-timeline-item>
        <hs-timeline-item>Item 3</hs-timeline-item>
      </hs-timeline>`
    );
    snap(
      'all the attributes set',
      `<hs-timeline alternate loading>
        <hs-timeline-item left type="brand">Item 1</hs-timeline-item>
        <hs-timeline-item last>Item 2</hs-timeline-item>
        <hs-timeline-item loading>Item 3</hs-timeline-item>
      </hs-timeline>`
    );
  });
});
