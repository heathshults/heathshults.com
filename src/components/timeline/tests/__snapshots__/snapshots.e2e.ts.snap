// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timeline renders all the attributes set 1`] = `"<blaze-timeline alternate=\\"\\" loading=\\"\\" class=\\"hydrated\\"><ol class=\\"hs-timeline hs-timeline--alternate hs-timeline--loading\\"><blaze-timeline-item left=\\"\\" type=\\"brand\\" class=\\"hydrated\\"><li class=\\"hs-timeline-item hs-timeline-item--brand hs-timeline-item--left\\"><div class=\\"hs-timeline-item__body\\">Item 1</div></li></blaze-timeline-item><blaze-timeline-item last=\\"\\" class=\\"hydrated\\"><li class=\\"hs-timeline-item hs-timeline-item--last\\"><div class=\\"hs-timeline-item__body\\">Item 2</div></li></blaze-timeline-item><blaze-timeline-item loading=\\"\\" class=\\"hydrated\\"><li class=\\"hs-timeline-item hs-timeline-item--loading\\"><div class=\\"hs-timeline-item__body\\">Item 3</div></li></blaze-timeline-item></ol></blaze-timeline>"`;

exports[`timeline renders with items 1`] = `"<blaze-timeline class=\\"hydrated\\"><ol class=\\"hs-timeline\\"><blaze-timeline-item class=\\"hydrated\\"><li class=\\"hs-timeline-item\\"><div class=\\"hs-timeline-item__body\\">Item 1</div></li></blaze-timeline-item><blaze-timeline-item class=\\"hydrated\\"><li class=\\"hs-timeline-item\\"><div class=\\"hs-timeline-item__body\\">Item 2</div></li></blaze-timeline-item><blaze-timeline-item class=\\"hydrated\\"><li class=\\"hs-timeline-item\\"><div class=\\"hs-timeline-item__body\\">Item 3</div></li></blaze-timeline-item></ol></blaze-timeline>"`;

exports[`timeline renders without items 1`] = `"<blaze-timeline class=\\"hydrated\\"><ol class=\\"hs-timeline\\"></ol></blaze-timeline>"`;
